### PR TITLE
Updated PR Template

### LINKED ISSUE #3489: PR template contains checkboxes that are never checked
**Describe the bug**
The PR template contains a bunch of check-boxes that never get checked, which makes open PRs appear unfinished.

**To Reproduce**
Look at the Pull Requests tab and verify that completed PRs show progress like "1 of 7": https://github.com/OpenMined/PySyft/pulls

**Expected behavior**
Finished PRs should show all tasks checked off. Specific tasks from the contributing guidelines should be listed in the checklist so that people can verify that they've done them without having to read the whole set of guidelines. The type of PR (new feature, bug fix, refactor, breaking change) should be identified with labels instead of check-boxes.

### PR DESCRIPTION
**Pull request by tkrsh**
## Updated PR Template 

The PR template contained a bunch of check-boxes that never got checked, which made open PRs appear unfinished.  

Update: 1)  Removed type of change checklist as it is supposed to be identified with a label.
              2)  Included few of the tasks from contribution in checklist.

Fixes #3489

## Type of change

Please mark the options that are relevant.

- [ ] Added/Modified tutorials
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

* [ ] I have added tests for my changes
* [x] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
* [ ] I have commented my code following [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,17 +7,15 @@ If your pull request closes a GitHub issue, then set its number below.
...
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 ## Checklist:
 
 * [ ] I have added tests for my changes
-* [ ] I did follow the [contribution guidelines](https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md)
+
+* [ ] I have Parametrized my test.
```
Comment (gmuraru) [inline .github/pull_request_template.md:15]: I think we should remove those checks.
This check should already be done by ```flake8``` or ```black```, right?

I think those stuff should already be caught if we merge [this PR](https://github.com/OpenMined/PySyft/pull/3434)

Comment (Jasopaum) [inline .github/pull_request_template.md:15]: This doesn't seem necessary for all PRs

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -7,17 +7,15 @@ If your pull request closes a GitHub issue, then set its number below.
...
+
+* [ ] I have ensured each import is within a seperate line.
```
Comment (gmuraru) [inline .github/pull_request_template.md:17]: Also, could you remove this :D

Comment (tkrsh) [reply .github/pull_request_template.md]: Done sir !

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -7,17 +7,11 @@ If your pull request closes a GitHub issue, then set its number below.
...
-Please mark the options that are relevant.
-
-- [ ] Added/Modified tutorials
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 
 ## Checklist:
 
 * [ ] I have added tests for my changes
```
Comment (gmuraru) [inline .github/pull_request_template.md:13]: Q: Could you also remove this.
Since there are changes which might not require this thing :D

Comment (tkrsh) [reply .github/pull_request_template.md:13]: Sure !

**Comment by tkrsh** [comment]
@gmuraru thank you for guiding me through my first PR !

**Pull request merged**